docs/commands/install.md

```diff
@@ -48,36 +48,10 @@ A number of applications are not currently available when using a private mirror
 
 - Maximo Assist
 - Maximo Health & Predict Utilities
-- Maximo Monitor
 - Maximo Predict
-- Maximo Safety
 - Maximo Visual Inspection
 
 
-SNO Support
```

Comment:
Why remove the SNO Support and Mustgather doc?

Reply:
- must-gather is out of date (replaced by dedicated new documentation https://ibm-mas.github.io/cli/commands/must-gather/
- sno section is a copy/paste of the airgap section plus a sentence that doesn't really add anything stating that we auto-detect SNO mode.